packit/distgit.py

```diff
@@ -372,18 +389,25 @@ def create_bodhi_update(
         if not koji_builds:
             # alternatively we can call something like `koji latest-build rawhide sen`
             builds_d = b.latest_builds(self.package_name)
+
+            builds_str = "\n".join([f" - {b}" for b in builds_d])
+            logger.info(f"Koji builds found: \n{builds_str}")
```

Comment:
please do debug here, this would print ~12 lines

Reply:
I thought it could be interesting to the user, but ok, I'll switch to debug.